src/r2mm/mods/ProfileImportExport.ts

```diff
@@ -49,8 +52,33 @@ async function saveDownloadedProfile(profileData: string): Promise<string> {
  * @returns string The path to the created file
  */
 async function downloadProfileCode(profileCode: string): Promise<string> {
-    const response = await ProfileApiClient.getProfile(profileCode);
-    return saveDownloadedProfile(response.data);
+    try {
+        let is404: boolean = false;
+        const response: AxiosResponse<string> = await retry(
+            () => ProfileApiClient.getProfile(profileCode),
+            5,
+            1000,
+            () => { return !is404 },
+            (e: any) => {
+                console.error(e);
+                if (e.message.startsWith("404")) {
+                    is404 = true;
+                }
+            },
+            true
+        );
+        return saveDownloadedProfile(response.data);
+    } catch (e: any) {
```

Comment:
You might want to consider getting rid of `any` here too. While the current implementation doesn't contain bugs, it increases a change that someone editing this creates one. If you expect the `e` be a regular `Error` here, something like this could suffice:

```
} catch (e: unkown) {
    if (e instance of Error && e.message === "Network Error") {
        ...
```